ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -57,37 +61,49 @@
     private String bucketName;
     private String directory;
     private Path outputPath;
-    private String prefix;
-    private boolean checksum;
+    private ImmutableList<String> prefixes;
     private Priority priority;
     private boolean sync;
-    private boolean force;
     private boolean discard;
     private int numberOfThreads;
 
+    private final static String PREFIX_SEPARATOR = ",";
+
     private final static ImmutableList<Option> requiredArgs = ImmutableList.of(BUCKET);
     private final static ImmutableList<Option> optionalArgs
             = ImmutableList.of(DIRECTORY, PREFIX, NUMBER_OF_THREADS,
-            DISCARD, PRIORITY, CHECKSUM, SYNC, FORCE);
+            DISCARD, PRIORITY, SYNC, FORCE);
 
     public GetBulk() {
     }
 
     @Override
     public CliCommand init(final Arguments args) throws Exception {
         processCommandOptions(requiredArgs, optionalArgs, args);
-        this.directory = args.getDirectory();
         this.bucketName = args.getBucket();
+        this.priority = args.getPriority();
+        this.numberOfThreads = args.getNumberOfThreads();
+
+        this.directory = args.getDirectory();
+        if (Guard.isStringNullOrEmpty(this.directory) || directory.equals(".")) {
+            this.outputPath = FileSystems.getDefault().getPath(".");
+        } else {
+            final Path dirPath = FileSystems.getDefault().getPath(directory);
+            this.outputPath = FileSystems.getDefault().getPath(".").resolve(dirPath);
+        }
+        LOG.info("Output Path = " + this.outputPath);
```

Comment:
Use the updated logging syntax: https://github.com/SpectraLogic/spectralogic.github.com/wiki/Java-Style-Guide#log-statements